up-l1/README.adoc

```diff
@@ -361,31 +415,52 @@ A `UTransport` implementation can use the `LocalUriProvider` to determine the uE
 
 A uEntity invokes this method to get its own authority.
 
+[.specitem,oft-sid="req~utransport-localuriprovider-getauthority~1",oft-needs="impl,utest"]
```

Comment:
this might require some transport specific design

```suggestion
[.specitem,oft-sid="req~utransport-localuriprovider-getauthority~1",oft-needs="dsn"]
```